src/utils/zcl_abapgit_convert.clas.abap

```diff
@@ -298,4 +314,44 @@ CLASS ZCL_ABAPGIT_CONVERT IMPLEMENTATION.
     GET BIT 8 OF iv_x INTO rv_bitbyte+7(1).
 
   ENDMETHOD.
+
+
+  METHOD date_to_string.
+
+    CONSTANTS: lc_date TYPE char1 VALUE 'D'.
+
+    CALL FUNCTION 'RRSV_IN_EX_CONVERT'
+      EXPORTING
+        i_chavl_int = iv_date
+        i_inttp     = lc_date
+      IMPORTING
+        e_chavl_ext = rv_date
+      EXCEPTIONS
+        OTHERS      = 0.
+
+    IF sy-subrc <> 0.
+      zcx_abapgit_exception=>raise( |Date to String Conversion failed. sy-subrc: { sy-subrc }| ).
+    ENDIF.
```

Comment:
I'm not familiar with setting OTHERS to 0, but shouldn't the sy-subrc check be pointless because it's always 0?

Reply:
Will change it to anything greater 0

Reply:
I think you can remove the `EXCEPTIONS` part altogether? The function itself does not raise any exceptions, so the `subrc` should always be `0`.

Regardless: I think you may have to change the function anyway, it seems to be a part of SAP_BW?

Reply:
Oh, ok I'll close it for now and search for another solution - thanks nonetheless for the review.

Reply:
Have changed the conversion methods - thanks for the review!